.github/workflows/test-python.yml

```
@@ -29,12 +37,13 @@ jobs:
           flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-      - name: Test with pytest (Linux and macOS)
-        if: matrix.os != 'windows-latest'
-        run: pytest
-      - name: Test with pytest (Windows)
-        if: matrix.os == 'windows-latest'
+      - name: Test with pytest
         shell: bash
-        run: GYP_MSVS_VERSION=2015 GYP_MSVS_OVERRIDE_PATH="C:\\Dummy" pytest
+        run: |
+          if [ "$RUNNER_OS" == "Windows" ]; then
+            GYP_MSVS_VERSION=2015 GYP_MSVS_OVERRIDE_PATH="C:\\Dummy" python -m pytest
+          else
+            python -m pytest
+          fi
```

```suggestion
          python -m pytest
```
`actions/setup-python` will ensure that `python` points to `matrix.python-version` on all platforms.

---

ah, cool!